src/elements/OCDrop.vue

```diff
@@ -1,12 +1,15 @@
 <template>
-  <div :uk-drop="$_ocDrop_props">
+  <div :id="$_ocDrop_id" :uk-drop="$_ocDrop_props" @click="$_ocDrop_close">
     <div v-if="$slots.default" class="uk-card uk-card-default uk-card-small uk-card-body">
       <slot />
     </div>
     <slot v-else name="special" />
   </div>
 </template>
 <script>
+import UiKit from "uikit"
+import { uniqueId as _uniqueId } from "lodash"
```

Comment:
is it really necessary to import lodash?

Reply:
I haven't found an alternative for the same function outside of downloading some other packages. One way could be using random number but there is the chance of generating two times the same number and then 💥 There is this function exported outside of lodash and distributed as own package but I'm not sure if we want to get it this way.

Reply:
Bloody uikit

Reply:
What about setting the ID via a property and leave it to the Developer who uses this component?
Default could be just oc-drop-default-id or Something ....

Reply:
Sounds reasonable... Will do :wink:

Reply:
Done